content/zh/docs/setup/additional-setup/compatibility-versions/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: 兼容版本
+description: 如何配置“兼容版本”，来将行为变更与版本解耦。
+weight: 36
+keywords: [profiles,install,helm]
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+Istio 的每个新版本都可能会出现一些有意而为的行为变更。
+这些变更可能是为了提高安全性、修复不正确的行为或以其他方式为用户改进 Istio。
+一般来说，这些类型的变更仅影响边缘情况。
+
+虽然从长远来看是有益的，但每次行为改变都会在升级过程中带来风险。
```

Comment:
```suggestion
虽然从长远来看是有益的，但每次行为变更都会引入升级风险。
```

Reply:
Applyed another suggestion for this line.